tests/integration/test_end_to_end.py

```diff
@@ -99,6 +107,15 @@ def test_end_to_end(capsys, fake_dns, fake_get_config, openstack_session):
         instance_name[-2:]
     )
 
+    # check ebs created
+    time.sleep(10)
```

Comment:
the launch method waits for the volume to be created. this sleep waits for the volume to be attached to the instance